app/src/lib/editors/win32.ts

```diff
@@ -302,6 +302,17 @@ const editors: WindowsExternalEditor[] = [
     displayNamePrefix: 'PhpStorm',
     publisher: 'JetBrains s.r.o.',
   },
+  {
+    name: 'Android Studio',
+    // registryKeys: registryKeysForJetBrainsIDE('Android Studio')
+    // Android Studio is based on JetBrains IDEs but I am not sure what to put here?
+    registryKeys: [
+      LocalMachineUninstallKey('Android Studio'),
+    ],
```

Comment:
Don't Android Studio's uninstall keys follow the same format as other Jet Brains IDEs? Something like `Android Studio 2021.4.1`

Reply:
I have looked for it but I couldn't be sure

Reply:
Did you find _any_ uninstall key for Android Studio? Can Desktop find Android Studio when built from this PR?

Reply:
I found a registry but like I said, I am not sure

Reply:
Cool! Could you show me what that key looks like so maybe I can help? 😄 🤞

Reply:
Ok. I am not in front of my pc now, I will message when I get home

Reply:
Hey @sergiou87, here is the photo of regedit:
![android](https://user-images.githubusercontent.com/76167420/156429836-e3233492-f796-444a-ae91-93e4d0eb3fd9.png)